src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -4751,7 +4751,11 @@ internal static string GetGetVerbText()
 param(
     [Parameter(ValueFromPipeline=$true)]
     [string[]]
-    $verb = '*'
+    $Verb = '*',
+
+    [Parameter(ValueFromPipeline=$true)]
+    [string[]]
+    $Group
```

Comment:
@lzybkr  Could you comment on the use of `enum` vs `validate set` ?
I attached the file with script and the resulting list of parameters whose type is enum (for Powershel 6 only).
[Test-Params.txt](https://github.com/PowerShell/PowerShell/files/618084/Test-Params.txt)